cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -298,6 +300,8 @@ protected void configure() {
     dsOpService.startAndWait();
     datasetService = injector.getInstance(DatasetService.class);
     datasetService.startAndWait();
+    namespaceService = injector.getInstance(NamespaceService.class);
```

Comment:
I was thinking AppFabricServer would be started, and that would start the namespace server.